### DISCUSSION TITLE
Are there any side-effects of using worklet in js thread functions

### DISCUSSION THREAD

**Discussion by swami-laxman** [software-mansion/react-native-reanimated#5100] [Q&A] [answered]
Let's say I have below function 

sum(a,b){
'worklet'
return a + b;
}

this function can be used on both the threads

But I want to know is it okay to call a 'worklet' function on js thread

can it cause memory leak or keep unwanted references in memory or other problems?

**Comment by tjzel** [comment, marked as answer]
It is absolutely safe to call this function on JS thread - no side effects will happen.
